src/openrct2/ride/RideData.cpp

```diff
@@ -729,7 +729,10 @@ const rct_ride_entry_vehicle CableLiftVehicle = {
     /* .effect_visual = */ 1,
     /* .draw_order = */ 14,
     /* .num_vertical_frames_override = */ 0,
-    /* .peep_loading_positions = */ 0
```

Comment:
Why did you remove this line?

Reply:
I don't know why... I never noticed because the code worked.

Reply:
can you put it back in it will still compile afaik.